presto-product-tests/bin/run_on_docker.sh

```diff
@@ -177,11 +177,15 @@ if [[ "$ENVIRONMENT" == "multinode" ]]; then
    PRESTO_SERVICES="${PRESTO_SERVICES} presto-worker"
 elif [[ "$ENVIRONMENT" == "multinode-tls-ldap" ]]; then
    PRESTO_SERVICES="${PRESTO_SERVICES} presto-worker-1 presto-worker-2 ldapserver"
+elif [[ "$ENVIRONMENT" == "multinode-tls-kerberos" ]]; then
+   PRESTO_SERVICES="${PRESTO_SERVICES} presto-worker-1"
```

Comment:
I would suggest you to have at least 2 worker nodes, to ensure that secured communication between the worker  nodes works fine as well as communication between the coordinator and the worker nodes.